src/main/java/no/nav/foreldrepenger/los/organisasjon/ansatt/AnsattTjeneste.java

```diff
@@ -96,7 +63,7 @@ public List<String> hentAvdelingerNavnForAnsatt(String ident) {
     }
 
     private static BrukerProfil mapTilDomene(AzureBrukerKlient.BrukerProfilResponse klientResponse) {
-        return new BrukerProfil(klientResponse.ident(), klientResponse.navn(), klientResponse.fornavnEtternavn(), klientResponse.epost(),
+        return new BrukerProfil(klientResponse.ident(), klientResponse.fornavnEtternavn(), klientResponse.fornavnEtternavn(), klientResponse.epost(),
```

Comment:
Skal vi droppe epost når vi først er i gang?

Reply:
Ja, kan droppe epost og axsys (gjør det etter frontend)

Det er et problem med fptilgang mot vtp (saksbehandlere lastes tilsynelatende ikke før man veksler token), men uten betydning for pipeline.